packages/manager/modules/vps/src/dashboard/migrate/index.scss

```diff
@@ -0,0 +1,6 @@
+oui-select-picker {
```

Comment:
can you scope this only to required module or select picker instance? this will be applied to all select pickers, across Manager, once this is loaded.